content/design-systems-for-developers/index.md

```diff
@@ -62,6 +62,7 @@ contributors: [
       detail: "UX Development at Shopify",
     },
 ]
+twitterShareText: "I’m learning about building design systems! They're great for scaling frontend code on large teams."
```

Comment:
I just made this up. Happy to edit or just merge and have you edit, but I wanted to have this come through in a PR so you could see it @domyen